policyengine_us/parameters/gov/states/mi/tax/income/deductions/standard/tier_two/amount/single_qualifying.yaml

```diff
@@ -0,0 +1,29 @@
+description: Michigan provides this standard deduction amount if you or your spouse are of qualifying age and qualifying condition.
+metadata:
+  label: Michigan standard deduction for qualifying seniors
+  period: year
+  unit: currency-USD
+  breakdown:
+    - filing_status
+  reference:
+    - title: Michigan Legal Code Section 206.30 - (9)(c)
+      href: http://legislature.mi.gov/doc.aspx?mcl-206-30
+    - title: Michigan 2022 MI-1040 Individual Income Tax FORMS AND INSTRUCTIONS
```

Comment:
comment: # Tax form represents the value as an additional amount while the legal code specifies the total amount